lib/server.js

```diff
@@ -85,51 +85,269 @@ module.exports = function (userApiClient, dataBroker, config) {
     };
 
 /**
- * @api {get} /status Request Service Status with software version
- * @apiName Status
- * @apiGroup Health-Check
- *
- * @apiSuccess {Object} status The status of the service
- * @apiSuccess {string} status.status The status of the service i.e. OK
- * @apiSuccess {string} status.version The semver version of the service
- * @apiSuccessExample {Object} Success-Response:
- *                  HTTP/1.1 200 OK
-                    { 
-                     'status': 'OK',
-                     'version': '0.2.1'
-                    }
+ * @swagger
+ * components:
+ *   securitySchemes:
+ *     tidepoolAuth:
+ *       type: apiKey
+ *       in: header
+ *       name: x-tidepool-session-token
+ *   responses:
+ *     InternalError:
+ *       description: An internal problem occured
+ *       content:
+ *         text/plain:
+ *           schema:
+ *             type: string
+ *     PageNotFound:
+ *       description: The requested document is not found
+ *       content:
+ *         text/plain:
+ *           schema:
+ *             type: string
+ *     Unauthorized:
+ *       description: The requester is not authorized to perform this request
+ *       content:
+ *         text/plain:
+ *           schema:
+ *             type: string
+ *   schemas:
+ *     ServiceStatus:
+ *       type: object
+ *       properties:
+ *         status: 
+ *           type: string
+ *         version:
+ *           type: string
+ *       example:
+ *         status: OK
+ *         version: 1.4.6
+ *     List:
+ *       type: object
+ *       additionalProperties:
+ *         schema:
+ *           oneOf:
+ *             - $ref: '#/components/schemas/Root'
+ *             - $ref: '#/components/schemas/Permission'
+ *       example:
+ *         subject:
+ *           root: {}
+ *         userX:
+ *           note: {}
+ *           preview: {}
+ *         userY:
+ *           note: {}
+ *           preview: {}
+ *     Root:
+ *       type: object
+ *       properties:
+ *         root:
+ *           type: object
+ *     Permission:
+ *       type: object
+ *       properties:
+ *         note: 
+ *           type: object
+ *         view:
+ *           type: object
+ */
+
+/**
+ * @swagger
+ * /status:
```

Comment:
Not sure we should document this route.
In the global context it will be documented as api.your-loops.com/status which does not exist...

Reply:
I agree with the global context. I am also wondering about the "local" context
What is the problem with documenting this route? @ydrisR

Reply:
I did replace the swagger JSDoc with comments about not documenting this route